.github/workflows/build.yml

```diff
@@ -0,0 +1,733 @@
+          apt-get update
+          apt-get install -y pixz \
+                             xz-utils \
+                             make \
+                             ninja-build \
+                             ccache \
+                             rpm
+
+      - name: Clone the anvill repository
+        uses: actions/checkout@v2
+        with:
+          path: ${{ steps.build_paths.outputs.REL_SOURCE }}/anvill
+          fetch-depth: 0
+
+      - name: Clone the remill repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'lifting-bits/remill'
+          path: ${{ steps.build_paths.outputs.REL_SOURCE }}/remill
+          fetch-depth: 0
+
+      - name: Select the remill version
+        run: |
+          remill_commit_id_path="${{ steps.build_paths.outputs.SOURCE }}/anvill/.remill_commit_id"
+
+          if [[ -f "${remill_commit_id_path}" ]] ; then
+            remill_commit_id=$(cat ${remill_commit_id_path})
+            echo "Using the following remill commit: ${remill_commit_id}"
+
+            cd ${{ steps.build_paths.outputs.SOURCE }}/remill
+            git checkout -b remill_commit_id "${remill_commit_id}"
+
+          else
+            echo "WARNING: Builds/tests will work against an unknown remill version"
+          fi
+
+      - name: Download the cxx-common ${{ matrix.cxxcommon_version}} package
+        shell: bash
+        id: cxxcommon_installer
+        working-directory: ${{ steps.build_paths.outputs.DOWNLOADS }}
+        run: |
+          folder_name="vcpkg_${{ matrix.image.name }}-${{ matrix.image.tag }}_llvm-${{ matrix.llvm }}_amd64"
+          archive_name="${folder_name}.tar.xz"
+
+          url="https://github.com/trailofbits/cxx-common/releases/download/${{ matrix.cxxcommon_version}}/${archive_name}"
+          local_path="${{ steps.build_paths.outputs.DOWNLOADS }}/${archive_name}"
+
+          if [[ ! -f "${local_path}" ]]; then
+            echo "Downloading: ${url}"
+            curl "${url}" -L -o "${local_path}"
+
+            ls -t ${{ steps.build_paths.outputs.DOWNLOADS }}/vcpkg_* | tail -n +2 | while read archive_file ; do
+              rm "${archive_file}"
+            done
+          fi
+
+          tar xf "${local_path}" \
+              -C "${{ steps.build_paths.outputs.INSTALL }}"
+
+          destination_path="${{ steps.build_paths.outputs.INSTALL }}/vcpkg"
+
+          mv "${{ steps.build_paths.outputs.INSTALL }}/${folder_name}" \
+             "${destination_path}"
+
+          echo "${destination_path}/installed/x64-linux/tools/llvm-11" >> $GITHUB_PATH
+
+          echo ::set-output name=PATH::${destination_path}
+
+      - name: Select the build job count
+        shell: bash
+        id: build_job_count
+        run: |
+          echo ::set-output name=VALUE::$(($(nproc) + 1))
+
+      - name: Configure remill
+        working-directory: ${{ steps.build_paths.outputs.BUILD }}
+
+        env:
+          CCACHE_DIR: ${{ steps.build_paths.outputs.CCACHE }}
```

Comment:
The reason it is only specified where it is used was to reduce scope so that it's not available in steps that don't need it. We can add it to the env if we want to move it